training/README.md

```diff
@@ -9,9 +9,6 @@ In order to run accelerated AI workloads, we've prepared [bootc](https://github.
 |-----------------|---------------------------------------------------------------------|
 | amd             | Create bootable container for AMD platform                          |
 | deepspeed       | DeepSpeed container for optimization deep learning                  |
-| cloud-amd       | Add cloud-init to bootable container for AMD platform               |
```

Comment:
Moving this is fine, but you should point add a section about building the images for different clouds and then link them together.

Reply:
Do we have anything in the README that talks about cloud images?  If not, we should.